bigframes/operations/semantics.py

```diff
@@ -39,7 +39,7 @@ def filter(self, instruction: str, model):
                 "The {food} is healthy."
 
             model:
-                A LLM model provided by Bigframes ML package.
+                A GeminiTextGenerator provided by Bigframes ML package.
```

Comment:
Could you please add an assert in the codes to check the model type is correct or not:
```
        if not isinstance(model, TextEmbeddingGenerator):
            raise TypeError(f"Expected a test embedding model but got: {type(model)}")
```
The semantics operators might required other model, such as `sem_index` needs an embedding model.

Reply:
Type check added.